src/main/cpp/Robot.cpp

```diff
@@ -36,6 +36,8 @@ const void EncoderValues() {
     frc::SmartDashboard::PutNumber("Arm Encoder Value", HatchIntake::getInstance()->GetEncoderValue());
     frc::SmartDashboard::PutNumber("Arm Pot Value", HatchIntake::getInstance()->GetArmPotValue());
     frc::SmartDashboard::PutNumber("Arm Pot Voltage", HatchIntake::getInstance()->GetArmPotVoltage());
+    frc::SmartDashboard::PutNumber("Elevator Encoder Value", Elevator::getInstance()->GetEncoderValue());
+    frc::SmartDashboard::PutBoolean("Elevator Banner Sensor", Elevator::getInstance()->IsFwdLimitSwitchClosed());
```

Comment:
`IsFwdLimitSwitchClosed` should be `IsRevLimitSwitchClosed`. The limit switch is going down.